source/modules/MEI.xml

```diff
@@ -1201,6 +1201,44 @@
       </valList>
     </content>
   </macroSpec>
+  <macroSpec ident="data.COURSENUMBER" module="MEI.stringtab" mode="add">
```

Comment:
Why `mode="add"` needed here?

Reply:
Also: Since it is declared as belonging to `MEI.stringtab` module, it should be moved there. Or change module name when it belongs here?

Reply:
We were orienting ourselves on other module implementations. Please kindly suggest improvements. Does it need to be mode="dt" since it's a data type definition?